### PR TITLE
:book: docs: add adopters file for existing ocm adopters

### PR DESCRIPTION
**Pull request by mikeshng**
## Summary

Add an adopters file for the existing adopters that correspond with the website https://open-cluster-management.io/

## Related issue(s)

Fixes #

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### Adopters
+
+Below is a list of companies and projects that have adopted Open Cluster Management(OCM). If you have been using OCM and your name is not on this list, send us a PR!
+
+- [Alibaba Cloud](https://www.alibabacloud.com/)
+- [Ant Group](https://www.antgroup.com/)
+- [Red Hat](https://www.redhat.com/)
+- [Tencent Cloud](https://cloud.tencent.com/)
+- [Xiao Hong Shu](https://www.xiaohongshu.com/)
```
Comment (qiujian16) [inline ADOPTERS.md:11]: @promacanthus would you confirm if it is ok?

File: ADOPTERS.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Who's using Open Cluster Management
+
+### Adopters
+
+Below is a list of companies and projects that have adopted Open Cluster Management(OCM). If you have been using OCM and your name is not on this list, send us a PR!
+
+- [Alibaba Cloud](https://www.alibabacloud.com/)
+- [Ant Group](https://www.antgroup.com/)
```
Comment (qiujian16) [inline ADOPTERS.md:8]: @ivan-cai Would you check if this is ok?

File: ADOPTERS.md (hunk 1 above)
Comment (promacanthus) [reply ADOPTERS.md:10]: > @promacanthus would you confirm if it is ok?

Yes, it's all good.

**Pull request merged**